docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -4,33 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>Let's call this a "URL template string" instead.</p><p> </p><p>It  might also be best to use valid python identifiers inside the curly braces and match the syntax from below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>